manifest.bioimage.io.yaml

```diff
@@ -29,6 +29,20 @@ config:
     - notebook
     - application
   url_root: https://raw.githubusercontent.com/deepimagej/models/master
+  docs: https://github.com/deepimagej/deepimagej-plugin/wiki
+  contact:
+    - name: Arrate Muñoz Barrutia
+      email: mamunozb@ing.uc3m.es
+      github: arratemunoz
+      affiliation:
+        - name: UC3M
+					url: https://www.uc3m.es/home
+		- name: Daniel Sage
```

Comment:
It looks like the format is not right. @cfusterbarcelo

Reply:
Don't know what happened when I copied. I think it's corrected now.